scripts/poppunk_extract_distances.py

```diff
@@ -59,12 +80,34 @@ def iterDistRows(refSeqs, querySeqs, self=True):
     with open(args.distances + ".pkl", 'rb') as pickle_file:
         rlist, qlist, self = pickle.load(pickle_file)
     X = np.load(args.distances + ".npy")
-
+    
     # get names order
-    names = iterDistRows(rlist, qlist, self)
+    r_names = isolateNameToLabel(rlist)
+    q_names = isolateNameToLabel(qlist)
+    names = listDistInts(r_names, q_names, self)
+
+    # parse distances from tree, if supplied
+    if args.tree is not None:
+        # only calculate if all v all
+        assert r_names == q_names, 'Using a phylogeny requires an all-v-all distance matrix'
+        # load tree
+        tree = dendropy.Tree.get(path = args.tree, schema = 'newick')
+        # calculate distance matrix
+        pdc = tree.phylogenetic_distance_matrix()
+        # dict for identifying nodes from names
+        tip_index = {}
+        for t in tree.taxon_namespace:
+            taxon_name = t.label.replace(' ','_')
```

Comment:
I guess this is due to some software that did the opposite switch at some point? Presumably this is ok because none of our sample names could ever have spaces in (just checking)